### DISCUSSION TITLE
Copying files...

### DISCUSSION THREAD

**Discussion by xenos-github** [dnGrep/dnGrep#810] [Ideas]
I'm using dnGrep to locate specific file types in order to create a minimal copy of useful files. Say, I might want all the pdf and xlsx files in a folder system exported (copied) to a memory stick.  So, I'm doing a search and using “More|Copy…”
For various reasons, I might want to repeat a copy action to update my copy. If I have squillions of files there, I don't want to have to erase it and start again. 
Would it be possible to have a “yes to all” type option in the copy dialogue box? As it is, it appears you have to click “yes” or “no” to answer a prompt asking if it's OK to overwrite each one of the files that is already there. 
A common approach to this is to have a checkbox for “don't ask me again” or “do this for all files”. 
I'm not using “More|Move…” right now, but a similar option there might be useful too. 

![Clipboard01](https://user-images.githubusercontent.com/71007867/214828216-435808c3-3af6-40df-8171-c2a5cefa9303.png)

**Comment by doug24** [comment]
Sounds like a good addition.
